tools/refresh-commands.sh

```diff
@@ -2,12 +2,28 @@
 
 TEMPDIR=$(mktemp -d)
 trap 'rm -rf "$TEMPDIR"' EXIT
-ORAS_COMMAND=$(./tools/install.sh ${TEMPDIR})
+
+#
+# This script generates command documentation by downloading the oras command
+# and parsing the help text to generate documentation. It iterates over the
+# `versions.json` file to determine the releases that are currently supported.
+#
+# If there is a fix to the help documentation in a patch version, use the
+# `versions-latest` file to specify the patch release for document generation.
+#
+# For example, If you are generating documentation for `1.2` and you want to
+# use `1.2.2` to generate the documentation, put a line in the `versions-latest`
+# file `1.2.2`.  If there is no line in the `versions-latest` that matches
+# the major and minor release `1.2`, then `1.2.0` will be used to generate
+# the documentation.
```

Comment:
@TerryHowe It seems the generator script can be tested by maintainers after this PRs merged into `main`, right?

Reply:
Yes, it will run with the `npm build`, but right now, the `npm build` does not fail if this fails.

Reply:
As soon as this merges, I'll create a PR with the latest updates.